docs/tls-authentication.md

```diff
@@ -0,0 +1,248 @@
+# Authentication
+
+To defend against impersonation attacks, as well as to maintain an encrypted communication
+between the wallet's frontend (a.k.a. _[daedalus](https://github.com/input-output-hk/daedalus)_)
```

Comment:
I would leave mention of daedalus out of these docs. Typical users that are following these steps are probably not using daedalus, but just accessing the API directly (e.g. exchanges).

Reply:
Perhaps I could reformulate to more general point of view, talking about _"clients"_ instead of daedalus for that it is what matters eventually.